### ISSUE TITLE
Enable "run" distro

### ISSUE THREAD

**Issue by kthoms** [operaton/operaton#314]
**Is your feature request related to a problem? Please describe.**
The Operaton Run modules are not part of the reactor. Module `distro/run` is commented out in `/pom.xml`.

**Describe the solution you'd like**
- Activate `distro/run` in `/pom.xml` and resolve build issues.
- Adjust `nightly_build.yml` to build and publish this distro.

**Comment by kthoms** [comment]
Enabling the `distro/run` module leads to a compile error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project operaton-bpm-run-qa-integration-tests: Compilation failure
[ERROR] /Users/karstenthoms/Development/projects/operaton/distro/run/qa/integration-tests/src/test/java/org/operaton/bpm/run/qa/ComponentAvailabilityIT.java:[119,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable swaggerUIAvailable
[ERROR]   Ort: Klasse org.operaton.bpm.run.qa.ComponentAvailabilityIT
```
![Bildschirmfoto 2024-12-05 um 12 42 40](https://github.com/user-attachments/assets/cb2e2d27-3a0a-4ce4-8884-2167b4cb3fc6)

**Comment by hauptmedia** [comment]
This could be related to https://github.com/operaton/operaton/pull/120

**Comment by hauptmedia** [comment]
This is actually related to the backported pull request, this whole test was removed here https://github.com/camunda/camunda-bpm-platform/commit/8fe087b1ee8941250c478bbcf4c3650f1eb40844#diff-730bce72fd850097409c2a7c43bd3bf16f33739fa3b48e626271aa8801b1ab25L118

I'll create a PR to fix this

**Comment by hauptmedia** [comment]
i think the original issue was not resolved by this, we mixed a bugfix here with the task of enabling the "run" distro

**Comment by javahippie** [comment]
Ah, it was closed automatically, because PR #322 stated `fixes #314`. If it should not close the issue, it should state `relates to #314`, didn't notice that. Thanks for reopening!
